pilates/activitysim/preprocessor.py

```diff
@@ -444,7 +444,7 @@ def _transit_skims(settings, transit_df, order, data_dir=None):
                                             measure_map[measure], order) * 100
                     
                 else:
-                    mtx = np.zeros((num_taz, num_taz))
+                    mtx = np.ones((num_taz, num_taz))
```

Comment:
this seems problematic, right @jdcaicedo251 ? Don't we want to keep Nulls set to 0?

Reply:
Hmm yeah, you're right, that shouldn't be in there any more, I'll change it back